data/projects.yml

```diff
@@ -451,3 +451,30 @@ project_list:
     github_user:
       - geealbers
     note: "While their 2020 cohort of scholars didn’t have an in-person conference to attend, the Museum Computer Network (MCN) instead offered them the opportunity to contribute to a unique publication that would focus on the current state of museum technology and sustainability. With ten essays in hand, Quire offered MCN a free and relatively easy way of creating the book and getting it online. MCN also made the title available on Amazon in print and e-book through the Kindle Direct Publishing service (https://kdp.amazon.com). A grant from Kress supported the scholars, editing, and the modest cost of printing and shipping author copies of the book, as well as copies to recent MCN Board members and supporters."
+
+  - id: "highpoint_editions"
+    src: /screenshots/_highpoint_editions.jpg
+    alt: "Highpoint Editions: A History & Catalogue, 2001–2021: Dennis Michael Jon, Jennifer L. Roberts, and Jill Ahlberg Yohe"
+    type: Collection Catalogue
```

Comment:
Let's go ahead and call this Catalogue Raisonné after all (to get the accent over the e press option+e and then e again)